src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -24,6 +24,10 @@ config:
     timeout: 400
     endpoint: 1
 
+    PIXIT.OO.MaxCommunicationTurnaround:
+        type: int16u
+        defaultValue: 45
```

Comment:
If all my comments get accepted then most probably this default value is too high.

Reply:
Updated.